client/src/pages/Dashboard.js

```diff
@@ -4,22 +4,16 @@ import React, { Component } from "react";
 import {LeftDrawer} from "../components/LeftDrawer";
 import {Copyright} from "../components/Copyright";
 
-
-
 //Material UI
-import { Grid, Typography } from '@material-ui/core';
-import Card from '@material-ui/core/Card';
-import CardMedia from '@material-ui/core/CardMedia';
-import CardContent from '@material-ui/core/CardContent';
-import PeopleAltIcon from '@material-ui/icons/PeopleAlt';
-import IconButton from '@material-ui/core/IconButton';
-import AssignmentTurnedInRoundedIcon from '@material-ui/icons/AssignmentTurnedInRounded';
-import NotificationsIcon from '@material-ui/icons/Notifications';
-import ErrorIcon from '@material-ui/icons/Error';
-import DoneIcon from '@material-ui/icons/Done';
-import AutorenewIcon from '@material-ui/icons/Autorenew';
-import Box from '@material-ui/core/Box';
-import Paper from '@material-ui/core/Paper';
+import {
```

Comment:
These new import statements are for migrating to material-ui v5 and react-router-dom v6.

In terms of actual code change, for `mui` most of the updates were due to the deprecation of `makeStyle`, and for `react-router-dom` it was the removal of the `history` state. This part was quite significant because of the extensive use of both APIs in our code.